web/packages/extension/assets/_locales/zh_TW/messages.json

```diff
@@ -26,6 +26,12 @@
     "settings_log_level_info": {
         "message": "資訊"
     },
+    "settings_log_level_debug": {
+        "message": "除錯"
+    },
+    "settings_log_level_trace": {
+        "message": "Trace"
```

Comment:
I guess this might be understood as is (some other strings are also kept in Latin script, such as "Flash", and "ActionScript"), and can always be translated (or even just transliterated at least) later...